packages/rest-api-client/src/client/RecordClient.ts

```diff
@@ -17,35 +17,37 @@ const ADD_RECORDS_LIMIT = 100;
 const UPDATE_RECORDS_LIMIT = 100;
 const DELETE_RECORDS_LIMIT = 100;
 
-type Entity = { code: string };
-type FileInformation = { fileKey: string };
+type EntityForParametor = { code: string };
+type FileInformationForParametor = { fileKey: string };
 type SubtableFieldValue =
   | number
   | string
   | string[]
   | null
-  | FileInformation[]
-  | Entity[];
+  | FileInformationForParametor[]
+  | EntityForParametor[];
+type SubtableForParametor = {
+  [fieldCode: string]: Array<
+    | {
+        value: SubtableFieldValue;
+      }
+    | {
+        id: number | string;
+        value?: SubtableFieldValue;
+      }
+  >;
+};
 type RecordForParameter = {
   [fieldCode: string]: {
     value:
       | number
       | string
       | string[]
       | null
-      | FileInformation[]
-      | Entity
-      | Entity[]
-      | {
-          [fieldCode: string]:
-            | {
-                value: SubtableFieldValue;
-              }
-            | {
-                id: number | string;
-                value?: SubtableFieldValue;
-              };
-        };
+      | FileInformationForParametor[]
+      | EntityForParametor
+      | EntityForParametor[]
+      | SubtableForParametor;
```

Comment:
@b4h0-c4t I found some misspellings. Please look for others.

```suggestion
      | FileInformationForParameter[]
      | EntityForParameter
      | EntityForParameter[]
      | SubtableForParameter;
```